logexp/cluster/pod.yaml

```diff
@@ -0,0 +1,54 @@
+# Template pod config for running the log exporter pod on a kubernetes node/master.
```

Comment:
Discussed in [this](https://github.com/kubernetes/test-infra/pull/2310#discussion_r109102384) comment.